examples/docs/src/layouts/MainLayout.astro

```diff
@@ -0,0 +1,123 @@
+---
+// import HeadCommon from "../components/HeadCommon.astro";
```

Comment:
with the workspace root change I was able to uncomment these and the site worked!